tests/src/services/users.service.test.ts

```diff
@@ -150,7 +183,7 @@ describe('UsersService tests', () => {
       });
     });
 
-    describe('Cancel the user B2B subscription', () => {
+    describe('cancelUserB2BSuscriptions()', () => {
```

Comment:
This goes directly against what we talked about in the last review. Talk about the use case, not about the function literally